code/gbq/run.py

```diff
@@ -181,6 +184,8 @@ def _get_test_quantile_predictions(model_config, run_config,
     
     train_seasons = df_train['season'].unique()
     
+    feat_importance = list()
```

Comment:
we will create a list of data frames with feature importance results, with one entry per bag and quantile level, and concatenate them later